tests/conftest.py

```diff
@@ -48,8 +48,10 @@ def assetdir():
 
 @pytest.fixture(scope="session")
 def launch_arguments(pytestconfig):
+    print(pytestconfig.getoption("--browser-channel"))
```

Comment:
do you think this has potential to get adopted in pytest-playwright? I think so.

Reply:
Yes, I think so.